python/letsql/config.py

```diff
@@ -0,0 +1,20 @@
+import pathlib
+from typing import Union
+
+from ibis.config import Config
+
+
+class Options(Config):
+    """LETSQL configuration options
+
+    Attributes
+    ----------
+
+    cache_default_path : str
+
+    """
+
+    cache_default_path: Union[str, pathlib.Path] = pathlib.Path("~/.local/share/letsql")
```

Comment:
if you don't call `.expanduser()` on the path, this is a relative path with the top level dir of `$(pwd)/~` and not `/home/$USER`

Reply:
Should it call absolute as well?